### DISCUSSION TITLE
Laser burning in only one direction

### DISCUSSION THREAD

**Discussion by whistlemeup08** [arkypita/LaserGRBL#1756] [Q&A]
I read this once but unable to find the correction.  The laser will only burn in left to right, not on return or from right to left.  There is a parameter that corrects this in the GRBL settings.

**Comment by StuartB4** [comment]
![tttttttttt](https://user-images.githubusercontent.com/16980026/153695474-ee0bd457-2320-4db4-82a4-7c090c11f937.png)
